### PR TITLE
Bumping actions/core and actions/github dependencies for GHES

### PR DESCRIPTION
**Pull request by pjquirk**
This adds support for running against GHES by using [v2.2.0](https://github.com/actions/toolkit/blob/master/packages/github/RELEASES.md#220) of `@actions/github`.  I also bumped `@actions/core` to 1.2.4 to fix error handling in the case an `Error` type is thrown rather than a `string`.

The build/pack steps seemed unreliable, so I updated the Typscript config to build to `./lib`, and `ncc` will then build out of that (this matches other actions like [actions/stale](https://github.com/actions/stale/blob/master/package.json#L6)).

Tested by pushing this version of the action to an "actions" org on the GHES instance.  I also needed to set `NODE_TLS_REJECT_UNAUTHORIZED = 0` temporarily in the action to get around my self-signed (or otherwise not trusted) server cert.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
   "main": "dist/index.js",
   "private": true,
   "scripts": {
-    "build": "ncc build src/main.ts",
+    "build": "tsc",
```
Comment (ericsciple) [inline package.json:40]: refer to checkout, `"tsc && ncc build`

Comment (ericsciple) [reply package.json:40]: simplifies, and runs quick enough

**Comment by pjquirk** [comment]
@anthonysterling Adding you since it looks like your team owns this action.

**Comment by jclem** [comment]
@pjquirk Can you elaborate on the tsc problem you were seeing? I haven't had any reliability problems there.

**Comment by pjquirk** [comment]
@jclem Before this PR, `tsc` would dump `main.js` into `dist/main.js`.   When I then ran `ncc build`, it didn't update the `index.js` file with the changes I had just made in the Typescript.  I didn't dig in too far, since it seemed wrong anyways to dump typescript output next to `ncc` output.

**Comment by jclem** [comment]
@pjquirk Running `tsc` manually shouldn’t be necessary—ncc has built-in TypeScript support already and compiled it using the version specified in our package.json file.

**Comment by jclem** [comment]
Thanks @pjquirk looks good! Glad we sorted out Webpack shenanigans together 😆

**Pull request merged**